src/replicache.ts

```diff
@@ -1237,6 +1258,201 @@ export class Replicache<MD extends MutatorDefs = {}> {
       return {result, ref};
     });
   }
+
+  private async _recoverMutations(): Promise<void> {
+    if (this._recoveringMutations) {
```

Comment:
TS is not able to reason about this async code. You need a `// @ts-ignore` here

Reply:
the issue is I haven't added so code to call `_recoverMutations` yet.